lab_1_keywords_tfidf/main.py

```diff
@@ -2,7 +2,47 @@
 Lab 1
 Extract keywords based on frequency related metrics
 """
-from typing import Optional, Union
+from typing import Optional, Union, Any
+from math import log
+from string import punctuation
+
+
+def correct_dict(variable: dict, type1: type, type2: type, empty: str) -> bool:
+    """
+    Checks the type of dict, keys and values
+    """
+    if isinstance(variable, dict) is True:
```

Comment:
simplify the condition. `is True` is not needed

Reply:
I think, I've fixed everything